libexec/plenv-commands

```diff
@@ -27,17 +27,11 @@ shopt -s nullglob
 { for path in "${paths[@]}"; do
     for command in "${path}/plenv-"*; do
       command="${command##*plenv-}"
-      if [ -n "$sh" ]; then
-        if [ ${command:0:3} = "sh-" ]; then
-          echo ${command##sh-}
-        fi
-      elif [ -n "$nosh" ]; then
-        if [ ${command:0:3} != "sh-" ]; then
-          echo ${command##sh-}
-        fi
-      else
-        echo ${command##sh-}
-      fi
+      case "$command" in
+        sh-*) [ -n "$nosh" ] && continue ;;
+           *) [ -n "$sh"   ] && continue ;;
+      esac
+      echo ${command##sh-}
     done
   done
-} | sort | uniq
+} | sort -u
```

Comment:
They should. [It was in POSIX at least by 2004](http://pubs.opengroup.org/onlinepubs/009695399/utilities/sort.html#tag_04_131_04) (I think it goes back further but it’s not easy to browse for historical POSIX editions).